workflows/run_dev_se.nf

```diff
@@ -0,0 +1,96 @@
+/***********************************************************************************************
+| WORKFLOW: PREPROCESSING ON SHORT-READ MGS DATA (EITHER SINGLE-END OR PAIRED-END) |
+***********************************************************************************************/
+
+import groovy.json.JsonOutput
+import java.time.LocalDateTime
+
+/***************************
+| MODULES AND SUBWORKFLOWS |
+***************************/
+
+include { RAW } from "../subworkflows/local/raw"
+include { CLEAN } from "../subworkflows/local/clean"
+include { PROCESS_OUTPUT } from "../subworkflows/local/processOutput"
+nextflow.preview.output = true
+
+/*****************
+| MAIN WORKFLOWS |
+*****************/
+
+// Complete primary workflow
+workflow RUN_DEV_SE {
+    // Start time
+    start_time = new Date()
+    start_time_str = start_time.format("YYYY-MM-dd HH:mm:ss z (Z)")
+
+    // Prepare samplesheet
```

Comment:
Importing the samplesheet is becoming enough lines where I think it may make sense to just put this in a process, perhaps called "LOAD_SAMPLESHEET", and call it in the workflow. @willbradshaw thoughts?

Reply:
FWIW, I think this would be very good. Happy to create this process.

Reply:
Will said he's good with this, @simonleandergrimm feel free to do this as a subworkflow or a process. The subworkflow will probably be easier as you can write in groovy directly in the subworkflow, but if you can figure out how to do groovy in the process, that works as well.